src/components/group-common.js

```diff
@@ -61,15 +62,15 @@ const UserMenuContent = ({ onEdit, onDelete }) => {
 const menuCardSize = 20
 
 export const MemberCardHeaders = Utils.memoWithName('MemberCardHeaders', ({ sort, onSort }) => {
-  return div({ style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem' } }, [
-    div({ style: { flex: 1 } }, [
+  return div({ role: 'row', style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem' } }, [
```

Comment:
Are we at all worried about this semantically being different from the aria perspective than the workspace table?

Reply:
In theory yes, but in practice I doubt a user is going to really care that
much as long as they can find what they're looking for. My colleague agreed
when he suggested going with a list for this one.